### ISSUE TITLE
AC_Allele2 not integer

### ISSUE THREAD

**Issue by ypnngaa-py** [weizhouUMICH/SAIGE#254]
Hi,
Downloaded the latest version yesterday (installed through conda env). I think it is 0.38 (didn't use the script for 0.41).
Using the binary mode for step1 and step2 I provided the vcf file  and specified `--vcfField=GT`. The resulted allele count including a mixture of integer and float like this:

```
AC_Allele2
--
75
11
22
21.0278015
867.832947
31.0136681
10.2809238
28.5665474
151.401245
38.2020378
```

Any idea of what's going on? Thanks

**Comment by weizhou0** [comment]
Hi @ypnngaa-py, 

Are there any samples with missing genotypes? If so, you may IsDropMissingDosages=F, otherwise, the missing genotypes with be mean imputed. 

Thanks,
Wei

**Comment by weizhouUMICH** [comment]
We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
